package.json

```diff
@@ -72,7 +72,7 @@
     "whatwg-fetch": "2.0.3"
   },
   "scripts": {
-    "start": "NODE_PATH=src && node scripts/start.js",
+    "start": "NODE_PATH=src && node scripts/build.js",
```

Comment:
What? Why are we running the build script instead of the start script here?? Why this change?

Reply:
I didn't notice. Thanks for pointing that out